express/code/blocks/ax-columns/ax-columns.css

```diff
@@ -829,6 +830,17 @@ main .ax-columns.highlight.dark .columns-video:last-of-type {
     padding-right: 20px;
     word-break: auto-phrase;
   }
+
+  .section .ax-columns.minimum-padding {
+    padding-left: 16px;
+    padding-right: 16px;
+    text-align: left;
```

Comment:
Is the text alignment needed? Disabling appears to not change anything.

Also worth mentioning. If a language is a RTL reading language this will not let the flow change, needing a RTL specific adjustment.

It would be [better to use](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#values) `text-align: start`. That will set the direction correctly to LTR or RTL.